service/routes.py

```diff
@@ -207,6 +207,28 @@ def get_item(order_id, item_id):
     return make_response(jsonify(message), status.HTTP_200_OK)
 
 
+######################################################################
+# DELETE AN ITEM FROM AN ORDER
+######################################################################
+@app.route("/orders/<int:order_id>/items/<int:item_id>", methods=["DELETE"])
+def delete_item(order_id, item_id):
+    """
+    Delete an item from an order
+    """
+    app.logger.info("Request to delete an Item %s from Order with id: %s", item_id, order_id)
+    
+    # See if the order exists and abort if it doesn't
+    order = Order.find(order_id)
+    if order:
```

Comment:
You can directly check if item exists in db. No need to add an extra check for order